pkgs/applications/editors/vim/plugins/generated.nix

```
@@ -17975,5 +17975,17 @@ final: prev:
     meta.homepage = "https://github.com/jhradilek/vim-snippets/";
   };
 
+  tailwind-tools-nvim = buildVimPlugin {
+    pname = "tailwind-tools-nvim";
```

```suggestion
    pname = "tailwind-tools.nvim";
```
This should be the original name of the plugin.

---

OK fixing

---

Fixed